docs/deployment/RELEASE_CHANNELS.md

```
@@ -1,47 +1,42 @@
-## Image
-
-Primary production image:
-
-```
-ghcr.io/bbartling/openfdd-edge-rust
-```
-
-MCP sidecar (separate workflow):
+## Images
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)